target/getmail/getmail-service.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# shellcheck source=../scripts/helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+
+# Directory, where "oldmail" files are stored.
+# getmail stores its state - its "memory" of what it has seen in your POP/IMAP account - in the oldmail files.
+GETMAIL_DIR=/var/lib/getmail
+
+# Kill all child processes on EXIT.
+# Otherwise 'supervisorctl restart getmail' leads to zombie 'sleep' processes.
+trap 'pkill --parent $$' EXIT
+
+function _syslog_error() {
+  logger --priority mail.err --tag getmail "${1}"
+}
+
+function _stopService() {
```

Comment:
Nope. I just forgot to convert my personal style here to DMS "standards" 😉

Personally I use camelCase for functions and for variables (your comment above) no `{}`, only when strictly necessary. `$var` is less to type and looks much cleaner and familiar to other languages. Especially in cases like `${$}, ${@}, ${1}`.. 

The same goes for using a `[[ ]]` pair per condition (your comment belog), instead of putting all conditions in just one `[[ ]]` pair. Sometimes it feels, like we purposely avoid the benefit a solutions provide. `[ $foo == "bar" && $bar == "foo" ]` is not posible, but it works with `[[  ]]`

If there had been a style discussion about the mentioned things in the past, I would definitely had vote against it 😆

EOR (End of rant^^).